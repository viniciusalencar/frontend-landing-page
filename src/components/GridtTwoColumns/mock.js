export default {
  title: 'Gridt Two Columns',
  text: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa animi quae autem officia molestiae aspernatur consectetur nam architecto ratione! Odio optio inventore veniam consequuntur, sed neque. Fugiat totam voluptatibus minima?`,
  srcImg: 'assets/images/javascript.svg',
};
