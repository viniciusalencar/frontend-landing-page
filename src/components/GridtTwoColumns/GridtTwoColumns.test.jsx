import { screen } from '@testing-library/react';
import { GridtTwoColumns } from '.';
import { renderTheme } from '../../styles/render-theme';
import mock from './mock';

describe('<GridtTwoColumns />', () => {
  it('should render two columns grid', () => {
    const { container } = renderTheme(<GridtTwoColumns {...mock} />);
    expect(screen.getByRole('heading')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
