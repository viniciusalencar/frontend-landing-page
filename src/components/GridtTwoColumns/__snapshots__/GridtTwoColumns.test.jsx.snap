// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridtTwoColumns /> should render two columns grid 1`] = `
.c5 {
  color: #FFFFFF;
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c2 {
  display: grid;
  grid-template-columns: 1fr 2fr;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 3.2rem;
}

.c2 .c4 {
  margin-bottom: 4.0rem;
}

.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c6 {
  font-size: 2.4rem;
}

@media (max-width:768px) {
  .c5 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c2 {
    grid-template-columns: 1fr;
    text-align: center;
  }
}

@media (max-width:768px) {
  .c3 {
    margin-bottom: 3.2rem;
  }
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <h1
            class="c4 c5"
          >
            Gridt Two Columns
          </h1>
          <p
            class="c6"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa animi quae autem officia molestiae aspernatur consectetur nam architecto ratione! Odio optio inventore veniam consequuntur, sed neque. Fugiat totam voluptatibus minima?
          </p>
        </div>
        <div
          class=""
        >
          <img
            alt="Gridt Two Columns"
            class=""
            src="assets/images/javascript.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
